vividus-bdd-engine/src/main/java/org/vividus/bdd/model/RunningStory.java

```diff
@@ -53,6 +53,11 @@ public boolean isFailed()
         return failed;
     }
 
+    public void setFailed(boolean failed)
```

Comment:
we shouldn't add setters just for unit tests, `fail ` method can be used instead